src/stories/Jobs/CanViewAllJobs/endpoint.spec.js

```diff
@@ -0,0 +1,94 @@
+const contextClassRef = requireUtil("contextHelper");
+const randomUser = requireUtil("randomUser");
+const knex = requireKnex();
+const httpServer = requireHttpServer();
+const JobsRepo = requireRepo("jobs");
+
+describe("Test API Jobs/CanViewAllJobs", () => {
+  beforeAll(async () => {
+    contextClassRef.user = randomUser();
+    contextClassRef.headers = {
+      Authorization: `Bearer ${contextClassRef.user.token}`,
+    };
+  });
+
+  it("dummy_story_which_will_pass", async () => {
```

Comment:
same feedback as previous story.

Reply:
@ShubhamJoshi-1994 `dummy_story_which_will_pass` I'll need to change the proper story name instead of `dummy` text, right?